src/middleware.ts

```diff
@@ -46,7 +46,24 @@ export type NamedSet<T extends State> = {
 export const devtools =
   <S extends State>(
     fn: (set: NamedSet<S>, get: GetState<S>, api: StoreApi<S>) => S,
-    prefix?: string
+    options?: {
```

Comment:
```suggestion
    options?: string | {
```
How about keep supporting the previous behavior so that we can avoid breaking changes?

Reply:
Thank you, yes that is a great suggestion.